pydnstest/scenario.py

```diff
@@ -781,6 +792,8 @@ def parse_entry(op, args, file_in, in_entry=False):
             out.set_edns(args)
         elif op == 'REPLY' or op == 'FLAGS':
             out.set_reply(args)
+        elif op == 'OPCODE':
```

Comment:
Support for explicit opcode specification this was merged into master just now: https://gitlab.labs.nic.cz/knot/deckard/merge_requests/60.